src/core/server/saved_objects/migrationsv2/model.ts

```diff
@@ -611,6 +685,8 @@ export const model = (currentState: State, resW: ResponseType<AllActionStates>):
         pitId: res.right.pitId,
         lastHitSortValue: undefined,
         hasTransformedDocs: false,
+        corruptDocumentIds: [],
```

Comment:
Initialize placeholders to collect transformation issues. We check the length of these through the migration.